docs/quick-start/transmitters/siyifm30.md

```diff
@@ -57,54 +57,34 @@ JR Module Pin | FR Mini Pin | Description
 | GND | GND | Use either the GND pin on the pinheader (outermost pin) or the GND pad on the programming header.
 | SPORT | TX2 | Use the TX2 pad on the receiver and remove the 4.7K pullup resistor. Without removing the resistor, the module will work okish as a transmitter, but firmware updates through OpenTX will fail with NoSync
 
+<figure markdown>
 ![FR Mini RX as TX Wiring](https://cdn.discordapp.com/attachments/738450139693449258/864205774251229224/frmini-rx-as-tx-wiring.jpg)
+</figure>
+
+### Flashing via STLink
 
-### FR Mini RX as TX Initial Flashing
+- Target: `FM30_RX_MINI_AS_TX_via_STLINK`
 
-The only way to flash the FR Mini is using STLINK. Note this is a one-way process, there is no returning to the stock firmware after flashing. The flashing procedure is similar to the FM30 TX module flashing listed above but using these pads on the receiver. Use the `FM30_RX_MINI_AS_TX_via_STLINK` target for initial flashing.
+!!! note
+    The only way to flash the FR Mini is using STLINK. This is a one-way process, there is no returning to the stock firmware after flashing. 
+    
+The flashing procedure is similar to the FM30 TX module flashing listed above but using these pads on the receiver. Use the `FM30_RX_MINI_AS_TX_via_STLINK` target for initial flashing.
 
+<figure markdown>
 ![FR Mini Pads](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/siyi/jupa/Siyi-12.JPG)
+</figure>
 
-### FR Mini RX as TX Updating
+### Updating via UART
+
+- Target: `FM30_RX_MINI_AS_TX_via_UART`
 
 Updates are done through OpenTX's built-in firmware flashing tool.
 
-* Build the firmware using Configurator and selecting the `FM30_RX_MINI_AS_TX_via_UART` target. The build process will generage a `firmware.elrs` file.
+* Build the firmware using Configurator and selecting the `FM30_RX_MINI_AS_TX_via_UART` target. The build process will generate a `firmware`.elrs` file.
 * Copy this file to the handset `FIRMWARE/` directory on the sd card.
 * Flash the firmware to the module using OpenTX
-  * Hold the MENU/SYS button on the handset to open the system menu
-  * Press PAGE to navigate to the SD card browser
-  * Scroll down to the FIRMWARE directory, and press ENTER
-  * Find `firmware.elrs` and long press the ENTER key to open the context menu
-  * Select `Flash Ext. ELRS` from the menu
-
-## Troubleshooting Flashing
```

Comment:
Same with this section. 

This tx module is unique that these troubleshooting steps only apply to this module. Putting this into the Troubleshooting page will only "clutter" the page, and these info can prove to be harder to find.

IMHO